build/Dockerfile

```diff
@@ -1,15 +1,18 @@
-FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:7.6
+FROM golang:1.10 AS builder
+ENV GOPATH=/go
+WORKDIR /go/src/github.com/kubevirt/hyperconverged-cluster-operator/
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build -o /hyperconverged-cluster-operator github.com/kubevirt/hyperconverged-cluster-operator/cmd/manager
 
+FROM centos:centos7
```

Comment:
Maybe we should be using the UBI here though now that it's released?

Reply:
I think that all the other kubevirt subprojects are going to move to ubi8

Reply:
Good thought... I will look into it